### PR TITLE
 Ghostfolio in Italiano #3587 

### PR DESCRIPTION
**Pull request by sghebuz**
Translate most of the new strings of Ghostfolio in Italiano #3587

### CODE REVIEW

File: apps/client/src/locales/messages.it.xlf [hunk 1]
```diff
@@ -796,7 +796,7 @@
       </trans-unit>
       <trans-unit id="a5e368dd1104a015e020de33a532e53e4d97525f" datatype="html">
         <source> Last Request </source>
-        <target state="new">Ultima richiesta</target>
+        <target state="translated"> Ultima richiesta </target>
```
Comment (dtslvr) [inline apps/client/src/locales/messages.it.xlf:799]: Removing spaces

File: apps/client/src/locales/messages.it.xlf [hunk 2]
```diff
@@ -5132,7 +5132,7 @@
       </trans-unit>
       <trans-unit id="80abe38d82bca467f47bbb65f821a115cf9dfdc1" datatype="html">
         <source> Are <x id="START_TAG_STRONG" ctype="x-strong" equiv-text="&lt;strong&gt;"/>you<x id="CLOSE_TAG_STRONG" ctype="x-strong" equiv-text="&lt;/strong&gt;"/> ready? </source>
-        <target state="new"><x id="START_TAG_STRONG" ctype="x-strong" equiv-text="&lt;strong&gt;"/>Sei<x id="CLOSE_TAG_STRONG" ctype="x-strong" equiv-text="&lt;/strong&gt;"/>pronto?</target>
+        <target state="translated"> <x id="START_TAG_STRONG" ctype="x-strong" equiv-text="&lt;strong&gt;"/>Sei<x id="CLOSE_TAG_STRONG" ctype="x-strong" equiv-text="&lt;/strong&gt;"/>pronto? </target>
```
Comment (dtslvr) [inline apps/client/src/locales/messages.it.xlf:5135]: Remove spaces

**Review by dtslvr — CHANGES_REQUESTED** [review]
Thanks for your contribution, @sghebuz! 😎
I have added a few comments.

File: apps/client/src/locales/messages.it.xlf [hunk 3]
```diff
@@ -6008,95 +6008,95 @@
...
+        <target state="translated">Software Open Source per la gestione della tua ricchezza</target>
         <context-group purpose="location">
           <context context-type="sourcefile">apps/client/src/app/pages/i18n/i18n-page.html</context>
           <context context-type="linenumber">14</context>
         </context-group>
       </trans-unit>
       <trans-unit id="metaKeywords" datatype="html">
         <source> app, asset, cryptocurrency, dashboard, etf, finance, management, performance, portfolio, software, stock, trading, wealth, web3 </source>
-        <target state="new"> app, asset, cryptocurrency, dashboard, etf, finance, management, performance, portfolio, software, stock, trading, wealth, web3 </target>
+        <target state="translated"> app, asset, cryptocurrency, dashboard, etf, finance, management, performance, portfolio, software, stock, trading, wealth, web3 </target>
```
[the remark this answers is not in the document]
Comment (sghebuz) [reply apps/client/src/locales/messages.it.xlf]: I tought they were not to be translated :-)
done!

File: apps/client/src/locales/messages.it.xlf (hunk 1 above)
Comment (sghebuz) [reply apps/client/src/locales/messages.it.xlf]: uhm ok done, I was just following the original spacing

File: apps/client/src/locales/messages.it.xlf (hunk 2 above)
Comment (sghebuz) [reply apps/client/src/locales/messages.it.xlf]: uhm ok done, I was just following the original spacing

File: apps/client/src/locales/messages.it.xlf (hunk 1 above)
Comment (dtslvr) [reply apps/client/src/locales/messages.it.xlf:799]: My bad, I have just reverted it.

**Review by dtslvr — APPROVED** [review]
Thanks again, @sghebuz 👍🏻
Ready to merge now.

**Pull request merged**